### ISSUE TITLE
0.12.x: Sometimes CI suspend

### ISSUE THREAD

**Issue by twoeths** [ChainSafe/lodestar#1003]
This happens in 0.12.x, sometimes I have to restart CI to got through the process. It got stuck at e2e step.
If I restart CI, I don't see the issue again.

**Comment by twoeths** [comment]
Some logs I found
```
@chainsafe/lodestar-cli: Error: Pubsub has not started
@chainsafe/lodestar-cli:     at LodestarGossipsub.unsubscribe (/home/runner/work/lodestar/lodestar/node_modules/libp2p-gossipsub/src/pubsub.js:317:13)
@chainsafe/lodestar-cli:     at unsubscribe (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/gossip.ts:165:21)
@chainsafe/lodestar-cli:     at Array.forEach (<anonymous>)
@chainsafe/lodestar-cli:     at Gossip.forEach [as unsubscribe] (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/gossip.ts:164:31)
@chainsafe/lodestar-cli:     at Gossip.unsubscribe [as unsubscribeFromAttestationSubnet] (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/gossip.ts:148:10)
@chainsafe/lodestar-cli:     at Timeout.unsubscribeFromAttestationSubnet [as _onTimeout] (/home/runner/work/lodestar/lodestar/packages/lodestar/src/sync/utils/attestation-collector.ts:75:25)
@chainsafe/lodestar-cli:     at listOnTimeout (internal/timers.js:551:17)
@chainsafe/lodestar-cli:     at processTimers (internal/timers.js:492:7) 
```

```
@chainsafe/lodestar-cli:   sync 2 dev nodes
@chainsafe/lodestar-cli: 2020-06-10 10:32:45  []                error: uncaughtException: listen EADDRINUSE: address already in use 127.0.0.1:30606
@chainsafe/lodestar-cli: Error: listen EADDRINUSE: address already in use 127.0.0.1:30606
@chainsafe/lodestar-cli:     at Server.setupListenHandle [as _listen2] (net.js:1313:16)
@chainsafe/lodestar-cli:     at listenInCluster (net.js:1361:12)
@chainsafe/lodestar-cli:     at doListen (net.js:1498:7)
@chainsafe/lodestar-cli:     at processTicksAndRejections (internal/process/task_queues.js:85:21) 

```

**Comment by twoeths** [comment]
```
@chainsafe/lodestar-cli:   5 passing (2m)
@chainsafe/lodestar-cli:   2 failing
@chainsafe/lodestar-cli:   1) e2e interop simulation
@chainsafe/lodestar-cli:        "after each" hook:
@chainsafe/lodestar-cli:      Uncaught Error: Pubsub has not started
@chainsafe/lodestar-cli:       at LodestarGossipsub.unsubscribe (/home/runner/work/lodestar/lodestar/node_modules/libp2p-gossipsub/src/pubsub.js:317:13)
@chainsafe/lodestar-cli:       at unsubscribe (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/gossip.ts:165:21)
@chainsafe/lodestar-cli:       at Array.forEach (<anonymous>)
@chainsafe/lodestar-cli:       at Gossip.forEach [as unsubscribe] (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/gossip.ts:164:31)
@chainsafe/lodestar-cli:       at Gossip.unsubscribe [as unsubscribeFromAttestationSubnet] (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/gossip.ts:148:10)
@chainsafe/lodestar-cli:       at Timeout.unsubscribeFromAttestationSubnet [as _onTimeout] (/home/runner/work/lodestar/lodestar/packages/lodestar/src/sync/utils/attestation-collector.ts:75:25)
@chainsafe/lodestar-cli:       at listOnTimeout (internal/timers.js:551:17)
@chainsafe/lodestar-cli:       at processTimers (internal/timers.js:492:7)
@chainsafe/lodestar-cli: 
@chainsafe/lodestar-cli:   2) sync 2 dev nodes
@chainsafe/lodestar-cli:        should connect 2 dev nodes until blocks found thru init & regular sync:
@chainsafe/lodestar-cli:      Uncaught Error: listen EADDRINUSE: address already in use 127.0.0.1:30606
@chainsafe/lodestar-cli:       at Server.setupListenHandle [as _listen2] (net.js:1313:16)
@chainsafe/lodestar-cli:       at listenInCluster (net.js:1361:12)
@chainsafe/lodestar-cli:       at doListen (net.js:1498:7)
@chainsafe/lodestar-cli:       at processTicksAndRejections (internal/process/task_queues.js:85:21)
```

**Comment by twoeths** [comment]
Should be resolved by #1007
